app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml

```diff
@@ -8,23 +8,41 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="ReorderButtonConfigurableProductTestStock" extends="AdminCacheValidationConfigurableProductSoldOutCustomStocks">
+    <test name="ReorderButtonConfigurableProductTestStock" extends="ReorderButtonConfigurableProductDefaultStock">
         <annotations>
             <stories value="Configurable Product Custom Stock."/>
             <title value="Reorder Button Visibility with Configurable Product on Custom Stock."/>
             <description value="Verify,reorder button is not visible after shipment created for order with configurable product on custom source."/>
-            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/2630678"/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/1676780"/>
```

Comment:
`StorefrontButtonReorderForTheOrderWithConfigurableProductOnTestStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest`

Reply:
Removed

Reply:
Will be removed in next PR